src/server/wtr-config.js

```diff
@@ -1,13 +1,23 @@
 import commandLineArgs from 'command-line-args';
 import { defaultReporter } from '@web/test-runner';
+import { headedMode } from './headed-mode-plugin.js';
 import { playwrightLauncher } from '@web/test-runner-playwright';
 import { visualDiff } from './visual-diff-plugin.js';
 
 const optionDefinitions = [
-	{ name: 'playwright', type: Boolean },
+	// @web/test-runner options
 	{ name: 'files', type: String, multiple: true, defaultOption: true },
 	{ name: 'group', type: String },
-	{ name: 'grep', alias: 'g', type: String, multiple: true }
+	{ name: 'manual', type: Boolean },
+	{ name: 'playwright', type: Boolean },
+	{ name: 'watch', type: Boolean },
+	// custom options
+	{ name: 'chromium', type: Boolean },
+	{ name: 'firefox', type: Boolean },
+	{ name: 'grep', alias: 'g', type: String },
+	{ name: 'filter', alias: 'f', type: String, multiple: true },
```

Comment:
`grep` now just forwards to the mocha config and works as it did before. `filter` (open to other names) will do what `grep` did previously: filter test files so you can run your tests in 2 seconds instead of 20.